vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -0,0 +1,20 @@
+module SecureRooms
+
+  module AccessRules
+
+    class BaseRule
+
+      def self.call(user, card_reader, accounts, selected)
```

Comment:
Do we think it might actually make more sense to pass the `product` rather than the `reader`?